shared/chat/inbox/index.native.js

```diff
@@ -183,21 +184,20 @@ class Inbox extends React.PureComponent<Props, State> {
       this.props.showSmallTeamsExpandDivider && (
         <BigTeamsDivider toggle={this.props.toggleSmallTeamsExpanded} />
       )
+    const buildTeam = this.props.showBuildATeam && <BuildTeam onBuildTeam={this.props.onBuildTeam} />
+    const HeadComponent = (
+      <ChatInboxHeader
+        filterFocusCount={this.props.filterFocusCount}
+        focusFilter={this.props.focusFilter}
+        onNewChat={this.props.onNewChat}
```

Comment:
lets copy onNewChat to its connector

Reply:
Discussed offline to leave this as is for the moment.